CHANGELOG.md

```diff
@@ -25,6 +25,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.3.1] - 2022-09-22
```

Comment:
Needs to go into a new 3.3.2 release since Eli's change to fix the oauthlib security vulnerability was version 3.3.1.

Reply:
I rebased against `main` to pull in the oauthlib fix.